.github/workflows/dismiss-stale-pr-reviews.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Dismiss Stale PR Reviews
+        if: ${{ !contains('OWNER,MEMBER,COLLABORATOR', github.event.pull_request.author_association) }}
```

Comment:
Definition of that enum: https://docs.github.com/en/graphql/reference/enums#commentauthorassociation